packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -284,6 +317,13 @@ typedef OverrideExtractFilesFromInputFunction = html.File Function(
   html.Element? input,
 );
 
+/// A function that extracts list of files from the file `input` passed in.
+@visibleForTesting
+typedef OverrideExtractMultipleFilesFromInputFunction = List<html.File>
+    Function(
+  html.Element? input,
+);
```

Comment:
Did you do `dart format` before submitting the PR? This looks odd :S

Reply:
i have done it now